File: tests/python/vote/voteThread.py
import requests
import json
import sys

jsonBody = {
	"thread_id": 1,
	"username": 'admin',
	"rating": 1
}

headers = {'Content-type': 'application/json', 'Accept': 'text/plain'}
res = requests.post('http://localhost:3000/voteThread', data = json.dumps(jsonBody), headers = headers)
print(res.json())

File: tests/nodejs/vote/voteThread.js
var request = require('request');

var jsonBody = {
	vote: 1,
	thread_id: 1
}

request({
	url: 'http://localhost:3000/voteThread',
	method: 'post',
	json: true,
	body: jsonBody
}, function (error, response, body) {
	console.log(response);
});